public/views/partials/molecules/formfield.liquid

```diff
@@ -49,6 +50,7 @@ metadata:
         content: Form label
       description_params:
         content: This is a FormField description
+        icon: Name
```

Comment:
In the Styleguide this would be an exact name of an icon.

Reply:
@ogabor82 could you please use github's suggestion feature to suggest a change? thanks!

Reply:
changed